src/hooks/twilio/useVideoRoomState.tsx

```diff
@@ -87,43 +83,25 @@ export const useVideoRoomState = (
 
     dismissVideoError();
 
-    // https://media.twiliocdn.com/sdk/js/video/releases/2.7.1/docs/global.html#ConnectOptions
-    await connect(token, {
-      name: roomName,
-      video: isActiveParticipant,
-      audio: isActiveParticipant,
-      enableDscp: true,
-    })
-      .then(setRoom)
-      .catch((error) => {
+    try {
+      joinChannel(userId, roomName, true, true);
```

Comment:
[code smell] Boolean trap

```suggestion
      joinChannel({
          userId,
          newChannelId: roomName,
          enableVideo: true,
          enableAudio: true
      });
```

Reply:
Resolved in 2da0e3efe